src/constants/networks/arbitrum.ts

```diff
@@ -19,8 +19,8 @@ const arbitrumInfo: NetworkInfo = {
   classicClient: createClient(
     'https://arbitrum-graph.kyberengineering.io/subgraphs/name/kybernetwork/kyberswap-exchange-arbitrum',
   ),
-  elasticClient: createClient('https://api.thegraph.com/subgraphs/name/kybernetwork/kyberswap-elastic-arbitrum-one'),
-  blockClient: createClient('https://api.thegraph.com/subgraphs/name/viet-nv/arbitrum-blocks'),
+  elasticClient: createClient('https://api.thegraph.com/subgraphs/name/viet-nv/kyberswap-elastic-arbitrum-one'),
```

Comment:
Is new subgraph breaking changes? if not deploy them on kybernetwork subgraph

Reply:
yep, breaking change, need to use my personal account first